UnitTests/UnitTests.UWP/App.xaml

```diff
@@ -0,0 +1,57 @@
+<Application x:Class="UnitTests.App"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Behaviors"
+             xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+             xmlns:helpers="using:UnitTests.Extensions.Helpers"
+             xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+             xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+             xmlns:ui="using:Microsoft.Toolkit.Uwp.UI"
+             xmlns:unitTestExtensions="using:UnitTests.Extensions"
+             RequestedTheme="Light">
+    <Application.Resources>
+        <muxc:XamlControlsResources>
+            <muxc:XamlControlsResources.MergedDictionaries>
+                <ResourceDictionary>
```

Comment:
Don't we need this?
```suggestion
                <ResourceDictionary>
                   <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
```

Reply:
This is the alternate syntax for including extra resources (as per their readme). It's what the `muxc:XamlControlsResources` part is on line 13. 🙂

Reply:
Missed that, thanks c: